tasks/module_4/task_1/disk_freespace.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Set default threshold value
+THRESHOLD=${1:-10}
+
+while true
+do
+    # Get available disk space in GB
+    FREE_SPACE=$(df -h / | awk 'NR==2 {print $4}' | cut -d'G' -f1)
```

Comment:
It'd be nice if you add an error handling. The scrip assumes that the df command will always return valid output, but if the command fails the script could produce unexpected behavior.

Reply:
Mmm I wonder if the `df -h` command fails will throw a `exit 0` and the process will get finished instead of producing a unexpected behavior.